lib/pythonx.ex

```diff
@@ -281,9 +281,17 @@ defmodule Pythonx do
   defmacro sigil_PY({:<<>>, _meta, [code]}, []) when is_binary(code) do
     %{referenced: referenced, defined: defined} = Pythonx.AST.scan_globals(code)
 
+    versioned_vars = __CALLER__.versioned_vars
```

Comment:
@josevalim let me know if you have any concerns with this approach :)

Reply:
Let’s use this function instead: https://hexdocs.pm/elixir/Macro.Env.html#has_var?/2

Reply:
Ohh perfect, I was almost certain we had this function, it's since v1.7 so I don't know how I missed it :D Thanks!